src/components/QuranReader/TranslationView/TranslationViewCell.tsx

```diff
@@ -64,7 +64,10 @@ const TranslationViewCell = ({ verse }: TranslationViewCellProps) => {
           </div>
```

Comment:
In addition to this change, can we modify the api call to not get the segment data. We no longer need it.

Reply:
Will create another PR for this 👍